packages/gatsby/package.json

```diff
@@ -126,6 +126,7 @@
     "moment": "^2.29.1",
     "multer": "^1.4.3",
     "node-fetch": "^2.6.6",
+    "node-html-parser": "^5.3.3",
```

Comment:
This should probably be in our `devDependencies`, as we don't use it outside of tests.

Reply:
Ignore this, just noticed this is how we grab head nodes in `cache-dir` 🤦